private/pkg/app/appcmd/docgenerator/docgenerator.go

```diff
@@ -0,0 +1,211 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docgenerator
```

Comment:
This doesn't follow the package naming convention here, and likely shouldn't be a sub-package `appcmd` anyways, but regardless, I'd just delete this package and move these functions to private functions in `appcmd`.